kep/42-podgroup-coscheduling/README.md

```diff
@@ -114,6 +115,12 @@ This extension pre-filters pods to save scheduling cycles. This is especially he
 
 For any pod that gets rejected, their pod group would be added to a backoff list and get retried until a TTL is met.
 
+#### PostFilter
+
+This extension will try to reject a group of pods if anyone of the group fails when `filter`
+
+> If more than 99% of pods pass `filter`, we will wait, otherwise we will reject the whole group.
```

Comment:
Does it read better?

```
If the gap to reach the quorum of a PodGroup is greater than 10%, 
we reject the whole PodGroup. Note that this plugin should be configured
as the last one of all PostFilter plugins.
```

Reply:
updated

Reply:
I meant to replace the whole section with the above stanza. (say don't need to specify "This extension ...").

Reply:
done